.github/workflows/test-e2e-cli.yml

```diff
@@ -9,75 +9,35 @@ on:
 
 env:
   GO_VERSION: 1.22.2
-  TEST_PARALLELISM: 8
+  TEST_PARALLELISM: 16
   TEST_PARALLELISM_PKG: 1
```

Comment:
Should we also change this? Is there any reason why e2e cli tests were running maximum 1 per package? Probably because it is a single package.

Reply:
Yes, there is only one package. 

Yes, you should increase `TEST_PARALLELISM`.

Reply:
It is done